js/src/lib.rs

```diff
@@ -58,6 +58,20 @@ impl JsHandle {
         panic!("unsupported doughnut version");
     }
 
+    /// Sign and return ed25519 signature
+    // #[wasm_bindgen(method, js_class = "Doughnut", js_name = signEd25519)]
+    #[allow(non_snake_case)]
+    #[wasm_bindgen(catch, method)]
+    pub fn signEd25519(&mut self, secret_key: &[u8]) {
+        if let Doughnut::V0(ref mut doughnut) = &mut self.0 {
+            doughnut.sign_ed25519(secret_key);
```

Comment:
Tried several ways, but it is still not working. Right now, it does not have compile issues, but it shows an error like the `sign_ed25519` function is unreachable when I am running unit test with `yarn test`, but the function does exist in bundled js file. Did some search around `unreachable` issue, but did not find anything useful. May need your help with looking into it @holygits  cc @alexsednz @KarishmaBothara

Reply:
the function in js is bound to `signEd25519`?

Reply:
Try `if let Doughnut::V0(d) = self {
            d.sign_ed25519(secret_key);`

Reply:
@holygits Yes, it is. Even I can print it out `signEd25519` is a function, it is still not reachable. Here is the error detail.

<img width="774" alt="Screen Shot 2020-06-25 at 4 26 19 PM" src="https://user-images.githubusercontent.com/1513326/85653172-d33b8e00-b700-11ea-9d30-de4a86738e19.png">

Reply:
@alexsednz It seems not like a valid code segment?